data/service/api/v1/datasets_data_create_provenance_test.go

```diff
@@ -61,15 +61,16 @@ func newTestReqAndDetails(clientID, userID, sourceIP string) (*rest.Request, req
 	if sourceIP != "" {
 		remoteAddr = sourceIP + ":1234"
 	}
+	token := newTestToken(clientID)
```

Comment:
How about a test covering this change? That is, no provenance is collected if the auth details do not have a token?

Reply:
Added.